config/settings-production.yml

```diff
@@ -15,6 +15,19 @@ buffy:
       hidden: true
     thanks:
       hidden: true
+    welcome:
+      if:
+        value_matches:
+          submission-type: "Standard|Estándar|Stats"
+      message: "Thanks for submitting to rOpenSci, our editors and @ropensci-review-bot will reply soon. Type `@ropensci-review-bot help` for help."
+      external_service:
+        url: http://138.68.123.59:8000/editorcheck
+        method: get
+        data_from_issue:
+          - repourl
+          - repo
+          - issue_id
```

Comment:
The `repo` and `issue_id` are sent by default, the `data_from_issue` param only refers to data to be extracted from the body of the issue. So for example the `repourl` value will be whatever is in the issue between `<!--repourl-->` and `<!--end-repourl-->` HTML comments.

Reply:
Thanks, you had previously clarified that, but also suggested it was okay to leave them there. They're all there just as an explicit identification of the parameters required by the external service, which makes it clearer and easier to cross-maintain these inter-dependent systems. Otherwise I'd have to add a note somewhere else that they're implicit passed, which would be more confusing that just stating them there.

Reply:
ah, ok. But `data_from_issue` will try to retrieve them from the body and if they are not present they will be nil. A better place to list them is under `query_params`:

```yaml
      external_service:
        url: http://138.68.123.59:8000/editorcheck
        method: get
        query_params:
          repo: optional_default_value
          issue_id: 
        data_from_issue:
          - repourl

```

Reply:
Aha, thanks - can you then just check the push i just made? The other `external_service` call also had all of those as `data_from_issue`. Does it all look okay now?

Reply:
Yes, I think that should work